packages/react/src/components/tile/Tile.tsx

```diff
@@ -3,20 +3,38 @@ import * as React from 'react';
 import {Svg, SvgNames} from '../svg';
 
 export interface TileProps {
-    title: string;
+    title?: string;
     svgName: SvgNames;
-    description: string;
-    href: string;
+    description?: string;
+    href?: string;
 }
 
-export const Tile: React.FunctionComponent<TileProps> = ({title, description, svgName, href}) => (
-    <a className="tile" href={href}>
+export const Tile: React.FunctionComponent<TileProps> = ({title, description, svgName, href}) => {
+    const tileIcon = (
         <div className="tile-icon">
             <Svg svgName={svgName} svgClass="icon" />
         </div>
+    );
+    const tileInfo = title && description && (
```

Comment:
I think that this condition would mean that you have to have a `title` and a `description` props set, right? Would it be better to check if one of the properties is set, and then check if the expected property is set (like you do below).
```suggestion
    const tileInfo = (title || description) && (
```

Reply:
mmhhh I think we don't want to have tile with only one or the other but always to have both a title and a description? :thinking: 

J'parle à travers mon chapeau, quote moi pas là-dessus :laughing:

Reply:
Yea good catch @sallainCoveo 👍